craft_providers/bases/buildd.py

```diff
@@ -521,6 +523,25 @@ def _setup_snapd(
                 details=errors.details_from_called_process_error(error),
             ) from error
 
+    def _setup_sudo(self, *, executor: Executor, deadline: Optional[float]) -> None:
+        """Install & configure sudo.
+
+        :param executor: Executor for target container.
+        :param deadline: Optional time.time() deadline.
+        """
+        _check_deadline(deadline)
+        try:
+            executor.execute_run(
+                ["apt-get", "install", "-y", "sudo"],
```

Comment:
why do we need sudo? we are already root.

Reply:
We are using sudo to standardize the behavior of exec'ing commands across the providers. Namely, the LXD-provided environment is quite a bit different than running `sudo cmd` for Multipass.  By using sudo for both, we get comparable environments.

Reply:
Removed sudo!